### ISSUE TITLE
web app needs to filter trips in dropdown by calendar start and stop date, not just day of week

### ISSUE THREAD

**Issue by unreasonableman** [cal-itp/graas#163]
For an example, see interactive grass report for Unitrans on Thu, March 10, "Clockwise to UCD MU @ 12:00 pm". The trip is recorded both by the baby shark and the ipad on bus #4000. Trip inference on the shark correctly assigns trip ID Q_13_1200, ipad reports Q_5_1200. Those trips are identical except for their calendar validity period. 

Unitrans is actually assigning their trip IDs very sensibly: Q is the shape name, 5 and 13 are the service IDs and 1200 is the time at the first stop. Service ID 13 is currently valid, ID 5 doesn't become valid until March 21.

Long story short, we shouldn't be adding trips with service IDs with non-matching start and stop dates to the dropdown.
